### PR TITLE
Few tweak suggestions

### PR DESCRIPTION
**Pull request by sapegin**
See my explanations in comments below.

### CODE REVIEW

File: manuscript/01_introduction.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Webpack isn't the only way to handle this problem possible and a collection of d
 
 ### How Webpack Changes the Situation?
 
-Webpack takes another route. It allows you to treat your project as a dependency graph. You could have an *index.js* in your project that pulls in the dependencies the project needs through standard `import` statements. You can refer to your style files and other assets the same way if you want.
+Webpack takes another route. It allows you to treat your project as a dependency graph. You could have an *index.js* in your project that pulls in the dependencies the project needs through standard `require` or `import` statements. You can refer to your style files and other assets the same way if you want.
```
Comment (sapegin) [inline manuscript/01_introduction.md]: I think even in 2017 many reader will not be familiar with ES6 syntax.

File: manuscript/02_webpack_compared.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ The popularity of Node.js and [npm](https://www.npmjs.com/), the Node.js package
 
 ## Task Runners and Bundlers
 
-Historically speaking, there have been many build systems. *Make* is perhaps the best known, and is still a viable option. Specialized *task runners*, such as Grunt, and Gulp were developed particularly JavaScript developers in mind. Plugins available through npm made both task runners powerful and extensible.
+Historically speaking, there have been many build systems. *Make* is perhaps the best known, and is still a viable option. Specialized *task runners*, such as Grunt, and Gulp were created particularly with JavaScript developers in mind. Plugins available through npm made both task runners powerful and extensible.
```
Comment (sapegin) [inline manuscript/02_webpack_compared.md]: Missed word and developed/developers very close to each other.

File: manuscript/02_webpack_compared.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ With Make, you model your tasks using Make-specific syntax and terminal commands
 
 ![Grunt](images/grunt.png)
 
-[Grunt](http://gruntjs.com/) went mainstream before Gulp. Especially its plugin architecture contributed towards its popularity. Plugins are often complex by themselves. As a result when configuration grows, it can become difficult to understand what's going on.
+[Grunt](http://gruntjs.com/) was the first popular task runner for front-end developers. Especially its plugin architecture contributed towards its popularity. Plugins are often complex by themselves. As a result when configuration grows, it can become difficult to understand what's going on.
```
Comment (sapegin) [inline manuscript/02_webpack_compared.md]: Since you’re discussing Gulp later, there is not much point in comparing Grunt to Gulp.

**Comment by sapegin** [comment]
I’d also add npm scripts. It‘s a popular alternative to Gulp/Grunt and I think it will be even more popular.

File: manuscript/02_webpack_compared.md (hunk 2 above)
Comment (bebraw) [inline manuscript/02_webpack_compared.md]: Makes sense.

File: manuscript/02_webpack_compared.md (hunk 3 above)
Comment (bebraw) [inline manuscript/02_webpack_compared.md]: Yup, true.

File: manuscript/01_introduction.md (hunk 1 above)
Comment (bebraw) [inline manuscript/01_introduction.md]: Yeah, better this way.

**Pull request merged**